Templates/project_collection/README.md

```diff
@@ -0,0 +1,232 @@
+# Instructions for filling out Project Collection Template
```

Comment:
Add instructions for what to do if there are multiple data collections in the project (e.g., add another tab)

Reply:
Currently adding another tab is not supported, but it could be made that way. Best would be a separate document for each data collection. I will make note of this.

Reply:
Adding instructions for this.